src/xdocs/anttask.xml.vm

```diff
@@ -101,7 +101,7 @@
           <td>file</td>
           <td>File to run checkstyle on.</td>
           <td>
-            One of either <i>file</i> or at least one nested <i>fileset</i>
+            One of either <i>file</i> or at least one nested <i>fileset</i> or <i>path</i>
```

Comment:
@romani Shouldn't each attribute be it's own separate row like we do for checks?
It is not like I can declare `path`s through the `file` attribute.
We should put this ant section under the control of our tests like we do for checks.